Adding multivareuler.m file & modifying task1.m

MrFlamadak opened the pull request:
multivareuler.m is a function that solves the differential equations of task1. task1.m is a program that calls on the function and plots the desired graph.

ruireng commented on Task1/multivareuler.m:
```diff
@@ -0,0 +1,34 @@
+  y0 = y0+yp*h;
+  x0 = x0+xp*h;
+  %preparation for next
+
```
I'd prefer that empty space didn't exist :)

MrFlamadak committed:
Adding multivareuler.m file & modifying task1.m

multivareuler.m is a function that solves the differential equations of
task1. task1.m is a program that calls on the function and plots the
desired graph.

MrFlamadak committed:
Removed newlines and added spaces
